test/e2e/sched/install/install_test.go

```diff
@@ -75,9 +75,21 @@ var _ = Describe("[Scheduler] install", func() {
 			Expect(err).NotTo(HaveOccurred())
 
 			By("checking the NumaResourcesScheduler Deployment is correctly deployed")
-			deploy, err := schedutils.GetDeploymentByOwnerReference(nroSchedObj.UID)
-			Expect(err).NotTo(HaveOccurred())
-			Expect(deploy.Status.ReadyReplicas).To(BeIdenticalTo(int32(1)))
+			const DeploymentCheckTimeout = 5 * time.Minute
+			const DeploymentCheckPollPeriod = 10 * time.Second
```

Comment:
C++-ism :) I'd rather avoid it but not a big deal

Reply:
on second thought we can also keep them
not a big deal

Reply:
maybe I should change the names to start with a lowercase letter to be more go-ish style

Reply:
I think yes, but it's minor.